src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -186,10 +186,28 @@ uiModules.get('apps/management')
       });
   };
 
+  this.isTimeBased = () => {
+    if (!this.formValues.timeFieldOption) {
+      // if they haven't choosen a time field, assume they will
+      return true;
+    }
+
+    // if timeFieldOption has a fieldName it's a time field, otherwise
+    // it's a way to opt-out of the time field or an indication that there
+    // are no fields available
+    return !!this.formValues.timeFieldOption.fieldName;
```

Comment:
I've seen `Boolean(` used in other places. Are we trying to standardize on one or the other?